assets/js/apitool-methods-ws.js

```diff
@@ -211,6 +211,17 @@ Request('ledger_entry - DirectoryNode', {
   }
 })
 
+Request('ledger_entry - NFT Page', {
+  description: "Returns an NFT Page object in its raw ledger format.",
+  link: "ledger_entry.html#get-offer-object",
```

Comment:
This link goes to the wrong example. I think you can leave out the `link:` attribute entirely, but it would be better to extend the [ledger_entry docs](https://xrpl.org/ledger_entry.html) with the format for this method and then link the new section.

Reply:
Agree - would be good to update ledger_entry docs with the format for this method.